src/lambda/commands/copyLambdaUrl.ts

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DefaultLambdaClient, LambdaClient } from '../../shared/clients/lambdaClient'
+import { LambdaFunctionNode } from '../explorer/lambdaFunctionNode'
+import globals from '../../shared/extensionGlobals'
+import { copyToClipboard as _copyToClipboard } from '../../shared/utilities/messages'
+import { addCodiconToString } from '../../shared/utilities/textUtilities'
+import { createQuickPick, QuickPickPrompter } from '../../shared/ui/pickerPrompter'
+import { isValidResponse } from '../../shared/wizards/wizard'
+import { FunctionUrlConfigList } from 'aws-sdk/clients/lambda'
+import { CancellationError } from '../../shared/utilities/timeoutUtils'
+
+export async function copyLambdaUrl(
+    node: Pick<LambdaFunctionNode, 'name' | 'regionCode'>,
+    client: LambdaClient = new DefaultLambdaClient(node.regionCode),
+    copyToClipboard = _copyToClipboard,
+    quickPickUrl = _quickPickUrl
+): Promise<void> {
+    const configs = await client.getFunctionUrlConfigs(node.name)
+
+    if (configs.length == 0) {
+        globals.window.showWarningMessage(
+            'No URL for Lambda function. [How to create URL.](https://docs.aws.amazon.com/lambda/latest/dg/urls-configuration.html)'
+        )
+        globals.window.setStatusBarMessage(addCodiconToString('circle-slash', 'No URL for Lambda function.'), 5000)
+    } else {
+        let url: string | undefined = undefined
+        const doQuickPick = configs.length > 1
```

Comment:
I'd like to hear your thoughts on this. I tried this way because I think it better explains 'what' the condition is trying to represent. The condition is now in English and I see it as easier to read since you don't need to guess what happens if the if statement is True.

 You don't know what happens when `if (configs.length > 1 )` is true.
But when you see, `if (doQuickPick)`, you know it will perform a quickpick if the statement is true.

I guess I could also do a comment, but let me know if you think this still isn't necessary. I have no strong feelings about this.

Reply:
> The condition is now in English and I see it as easier to read since you don't need to guess what happens if the if statement is True.

I think your overall reasoning about this makes perfect sense: we definitely want to make logic easy for others to read. A lot of this comes down to personal preference though I think there are some general guidelines that are applicable to most developers. In this case I'd argue that `doQuickPick` is _harder_ to read because it doesn't tell me anything more than what the logic already says:
```ts
        if (configs.length > 1) {
            url = await quickPickUrl(configs)
        }
```
Which reads as "if there's more than one config, prompt the user". This snippet stands perfectly well on its own IMO. Also keep in mind that code is not always read from top to bottom. Sometimes we want to know how things work starting at the end result.